src/components/SearchInput/SearchInput.module.css

```diff
@@ -0,0 +1,53 @@
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid black;
+  padding: 4px;
+  border-radius: 4px;
+  width: 100%;
+  height: inherit;
+}
+
+.container:focus-within {
+  outline: 1px solid black;
+}
+
+.container:focus-within .searchIcon {
+  opacity: 1;
+}
+
+.clearIcon {
+  border: 0;
+  background: none;
+  cursor: pointer;
+}
+
+.input {
+  outline: none;
+  border: none;
+  padding: 0 4px;
+  width: 100%;
+}
+
+.input::-webkit-search-cancel-button {
+  visibility: hidden;
+  pointer-events: none;
+}
```

Comment:
I'm seeing the default cancel button in safari, fwiw:

https://user-images.githubusercontent.com/157195/185189844-c64b98cd-9127-4b2e-8892-b0ae8f6089d5.mov

Reply:
Ah, good 👁️ . I'll take care of it!